frontend/src/app/word-models/related-words/related-words.component.ts

```diff
@@ -28,15 +32,37 @@ export class RelatedWordsComponent implements OnChanges {
 
     faCheck = faCheck;
 
-    constructor(private wordModelsService: WordmodelsService) { }
+    constructor(
+        route: ActivatedRoute,
+        router: Router,
+        paramService: ParamService,
+        private wordModelsService: WordmodelsService
+    ) {
+        super(route, router, paramService);
+    }
 
     ngOnChanges(changes: SimpleChanges) {
         if (changes.corpus || changes.queryText) {
             this.getData();
         }
     }
 
+    initialize() {
+        
+    }
```

Comment:
This has trailing spaces.

```suggestion
    initialize() {
    }
```